api/src/wfl/api/routes.clj

```diff
@@ -120,11 +120,18 @@
   "Top level exception handler. Prefer to use status and message
    from EXCEPTION and fallback to the provided STATUS and MESSAGE."
   [status message exception request]
-  {:status (or (:status (ex-data exception)) status)
-   :body {:message (or (.getMessage exception) message)
-          :exception (.getClass exception)
-          :data (ex-data exception)
-          :uri (:uri request)}})
+  (let [response {:status (or (:status (ex-data exception)) status)
+                  :body {:message (or (.getMessage exception) message)
+                         :exception (str (.getClass exception))
+                         :data (ex-data exception)
+                         :uri (:uri request)}}]
+    (if (>= (:status response) 500)
+      (do
+        (log/errorf "Server %s error at occurred at %s :"
+                    (:status response)
+                    (:uri (:body response)))
+        (logr/error exception (:body response))))
+    response))
```

Comment:
OK, but it would be cleaner if you created another handler that does this, then delegates to the first.

Reply:
Example:
https://github.com/broadinstitute/wfl/blob/c63fa0ca24324a504c3c294fcbe6664279d22396/api/src/wfl/api/routes.clj#L129

Reply:
Yes. I think we want a logging backstop handler and everything else close over `ex-handler` if you want.

Reply:
Good idea! I reversed the execution order from Ed's idea since I figure logging the response code and the user's input is helpful and we calculate that out in the backstop handler